OxfordBodleian/Juel-Jensen/BDLaethe39.xml

```diff
@@ -65,189 +61,789 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    በከመ፡ ይቤ፡ ጳውሎስ፡ እስመ፡ ደቂቅ፡ ተሳተፉ፡ ደመ፡ <add place="interlinear">ወ</add>ሥጋ። ውእቱኒ፡ ተሳተፈ፡ ምስሌሆሙ። 
+                                    በአዳም፡ ቅድመ፡ ይሰደድ፡ እምገነት። ወአመ፡ ይቤሉ፡ ባሕቱ፡ እግዚ
+                                </explicit>
+                                <note>The text ends abruptly due to loss of one or more leaves.</note>
+                            </msItem>
+                            <msItem xml:id="p2_i3">
+                                <locus from="89ra" to="90va"/>
+                                <title type="incomplete" ref="LIT6075DiscourseOnSoul"/>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    <locus target="89ra"/>
+                                    ስማዕ፡ ኦብእሲ፡ በቍኤታ፡ ለነፍስከ፡ እግዚአብሔር፡ ውእቱ፡ ፈጣሪከ፡ መሀረከ፡ ወ<add place="interlinear">ኢ</add>ገብረ፡ ለከ፡ በከመ፡ እከይከ፡ 
+                                    ወኢፈደየከ፡ በከመ፡ ኃጢአተከ፡ ውእቱሰ፡ ይጸንሐከ፡ እስከ፡ ትትኔሳሕ፡ እምእከይከ፡ ወትትመየጥ፡ ኀቤሁ። ስማዕ፡ ኦብእሲ፡
+                                    እንግርከ፡ ውእቱሰ፡ ፈጣሪ፡ እንዘ፡ ይሬኢ፡ ኀጢአተከ፡
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus target="90va"/>
+                                    ለኵሉ፡ ፍኅረት፡ <gap reason="illegible" unit="chars" quantity="3"/> ወብ<gap reason="illegible" unit="chars" quantity="1"/>፡ ለኵሉ፡ ሰብእ፡ 
+                                    ተ<gap reason="illegible" unit="chars" quantity="1"/>ሐት። ወተፍጻሜቱ፡ ለኵሉ፡ መ<gap reason="illegible" unit="chars" quantity="5"/>፡ ዓለመ፡ ለዓለመ፡ ዓለም፡ አሜን።
+                                </explicit>
+                            </msItem>
+                            
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Other">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>                                     
+                                    </support>
+                                    <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay in January 2020.</foliation>
+                                    <collation>
+                                        <list>
+                                            <item xml:id="q19">
+                                                <dim unit="leaf">3</dim>
+                                                <locus from="88r" to="90v"/>
+                                            </item>
+                                        </list>
+                                    </collation>
+                                    <condition key="deficient">
+                                        The quire originates from another codex and is currently held together very precariously by means of one stitch.
```

Comment:
It is the so called 'running stitch' :)

Reply:
Updated the record with this information! Then, you will be able to give more precise details on this, which I possibly overlooked.